packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -1840,4 +1866,33 @@ export default class TradeStore extends BaseStore {
     setIsDigitsWidgetActive(is_active: boolean) {
         this.is_digits_widget_active = is_active;
     }
+
+    /******************************** V2 Specific Logic ********************************/
+    setFavoriteIndicators(indicators: string[]) {
```

Comment:
`trade-store` is already too huge. It's better to avoid adding more to the trade-store. 
Can we create a new store for markets/symbols instead?